go.mod

```diff
@@ -90,22 +90,22 @@ require (
 	github.com/libp2p/go-ws-transport v0.5.0
 	github.com/miekg/dns v1.1.43
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/multiformats/go-multiaddr v0.4.0
+	github.com/multiformats/go-multiaddr v0.4.1
```

Comment:
plaintextv2 code point added https://github.com/multiformats/go-multiaddr/releases/tag/v0.4.1

Reply:
We probably don’t need to do this, unless this is pulled in by some other dependency update. On the other, it won’t hurt either.

Reply:
ya, we don't need it but figure for dependencies we control we should probably try and run with the latest so we can detect any issues